test/data/664.yml

```diff
@@ -0,0 +1,19 @@
+---
+:class: MemberPage
+:url: http://www.nass.gov.ng/mp/profile/664
+:to_h:
+  :id: '664'
+  :title: Hon.
+  :shorter_name: BELLO ABDULLAHI
+  :name: BELLO ABDULLAHI
+  :constituency: Okene/Ogori-Magogo
+  :state: Kogi State
+  :chamber: House of Representatives
+  :position: ''
```

Comment:
Again, it's worth making sure that every field is covered by at least one test case.